internal/hud/server/server_test.go

```diff
@@ -21,6 +24,16 @@ import (
 	"github.com/windmilleng/tilt/pkg/model"
 )
 
+var originalWD string
+
+func init() {
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	originalWD = wd
```

Comment:
Hmm why do we need to do this? Does the HUD server test muck with the current working directory?

Reply:
Haha, as far as I know we were just trying to read the testdata dump with all the JSON, and found this snippet in some SO answer.